contracts/VestingSimple.sol

```diff
@@ -0,0 +1,103 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.5.8;
+pragma experimental ABIEncoderV2;
+
+import "./Token.sol";
+import "../lib/dappsys/auth.sol";
+import "../lib/dappsys/math.sol";
+import "../lib/dappsys/erc20.sol";
+
+
+contract VestingSimple is DSMath, DSAuth {
+
+  event GrantAdded(address recipient, uint256 amount);
+  event GrantClaimed(address recipient, uint256 amount);
+
+  Token public token;
+
+  uint256 public base;
+  uint256 public period;
+
+  bool public isActive;
+  uint256 public startTime;
+
+  struct Grant {
+    uint256 amount;
+    uint256 claimed;
+  }
+
+  mapping (address => Grant) public grants;
+
+  modifier inactive() {
+    require(!isActive, "vesting-simple-already-active");
+    _;
+  }
+
+  modifier active() {
+    require(isActive, "vesting-simple-not-active");
+    _;
+  }
+
+  constructor(address _token, uint256 _base, uint256 _period) public {
+    token = Token(_token);
+    base = _base;
+    period = _period;
+  }
+
+  function activate() public auth inactive {
+    isActive = true;
+    startTime = now;
+  }
+
+  function withdraw(uint256 _amount) public auth {
```

Comment:
I added this as a security measure, just in case there's some bug somewhere and all the tokens would be locked. Just like we're trusting ourselves to add the correct amount of tokens, we trust ourselves not to remove them. But I suppose it doesn't matter since we're minting CLNY anyway, so it's no great loss if something goes wrong.

Reply:
Okay. We should just check with Jack that this is okay to have.

Reply:
Jack is alright with this. Ultimately this isn't trustless as much as us setting up a trusted mechanism for people to receive tokens form us.